File: JavaBasic/180_index_of_elem.java
// 180	180_index_of_elem.java	Find the index of a specific element in an array using indexOf().
import java.util.*;
public class _180_index_of_elem {
    public static void main(String[] args) {
        List<Integer> list = Arrays.asList(15,30,45,60,75);
        int elem = 45;
        System.out.println("Index of " + elem + ": " + list.indexOf(elem));
    }
}

File: JSBASIC/180_index_of_elem.js
// 180	180_index_of_elem.js	Find the index of a specific element in an array using indexOf().

let arr = [15, 30, 45, 60, 75];
let elem = 45;
let index = arr.indexOf(elem);
console.log("Array:", arr);
console.log("Searching for:", elem);
console.log("Index:", index !== -1 ? index : "Not found");
